yt/docs/en/_includes/user-guide/data-processing/spyt/cluster/configuration.md

```diff
@@ -49,6 +49,18 @@ spark-launch-yt \
   --spark-cluster-version '1.36.0'
 ```
 
+### Spark configuration
+
+When using `spark-launch-yt` to configure a cluster, `--params '{"spark_conf"={...};}` is available as an option:
```

Comment:
I think it should be explained why it is needed to configure a cluster via this option.